standards-catalogue/source/layouts/standards_table.erb

```diff
@@ -16,6 +16,8 @@
             <td nowrap><strong><%= display_organisation(value) %></strong> </td>
           <% elsif key == 'status' %>
             <td nowrap><strong><%= display_status(:standard, value) %></strong> </td>
+          <% elsif key == 'licence_id' %>
+            <%= display_licence(value) %>
```

Comment:
Will remove all these if statements when I fix all the other frontmatter content